README.md

```diff
@@ -94,6 +109,28 @@ import { ReactSelector } from 'testcafe-react-selectors';
 const item = ReactSelector('TodoItem').withKey('HighPriority');
 ```
 
+#### Selecting components by display name
+
+You can select elements by the component's [displayName](https://reactjs.org/docs/react-component.html#displayname).
+
+For instance, consider the `TodoList` component whose `displayName` class property is specified as follows:
+
+```js
+class TodoList extends React.Component {
+  // ...
+}
+
+TodoList.displayName = 'todo-list-display-name';
```

Comment:
Since `todo-list-display-name` is kind of a class name, I wouldn't add dash in `displayName`
```js
TodoList.displayName = 'TodoList';
```